hazelcast/src/main/java/com/hazelcast/jet/impl/LightMasterContext.java

```diff
@@ -116,7 +118,7 @@ public LightMasterContext(NodeEngine nodeEngine, JobCoordinationService coordina
         dag.iterator().forEachRemaining(vertices::add);
         Map<MemberInfo, ExecutionPlan> executionPlanMapTmp;
         try {
-            executionPlanMapTmp = createExecutionPlans(nodeEngine, members, dag, jobId, jobId, config, 0, true);
+            executionPlanMapTmp = createExecutionPlans(nodeEngine, members, dag, jobId, jobId, config, 0, true, null);
```

Comment:
We won't check the permissions for SecuredFunctions for light jobs? The `subject` (the last argument) is always `null`